settings.gradle.kts

```diff
@@ -7,7 +7,6 @@ include("scavenger-schema")
 include("scavenger-agent-java")
 include("scavenger-agent-python")
 include("scavenger-model")
-include("scavenger-old-agent-java")
```

Comment:
Why?

Reply:
I'm sorry. I turned back this line.
There was a error when building `scavenger-old-agent-java` in my device,
so I temporarily removed this line and run test.

Reply:
what is the error message?
please tell me.. I think it's better to include old-agent-java conditionally.

Reply:
There are two kinds of errors depending on the JDK versions. (M1 Mac)

#### JDK 1.8

(azul, adoptium)

```
Build file '/Users/user/Documents/GitHub/scavenger/scavenger-old-agent-java/build.gradle.kts' line: 4

An exception occurred applying plugin request [id: 'com.palantir.git-version', version: '0.15.0']
> Failed to apply plugin 'com.palantir.git-version'.
   > Could not create plugin of type 'GitVersionPlugin'.
      > Could not generate a decorated class for type GitVersionPlugin.
         > org/eclipse/jgit/lib/Repository has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```

#### JDK 11, 17

(adoptium only)

```
Build file '/Users/user/Documents/GitHub/scavenger/scavenger-old-agent-java/build.gradle.kts' line: 112

No compatible toolchains found for request filter: {languageVersion=7, vendor=any, implementation=vendor-specific} (auto-detect true, auto-download true)

* Try:
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Exception is:
org.gradle.jvm.toolchain.internal.NoToolchainAvailableException: No compatible toolchains found for request filter: {languageVersion=7, vendor=any, implementation=vendor-specific} (auto-detect true, auto-download true)
	at org.gradle.jvm.toolchain.internal.JavaToolchainQueryService.lambda$noToolchainAvailable$5(JavaToolchainQueryService.java:96)
...
```

Reply:
> An exception occurred applying plugin request [id: 'com.palantir.git-version', version: '0.15.0']

This error will be fixed by #39.

> No compatible toolchains found for request filter: {languageVersion=7, vendor=any, implementation=vendor-specific} (auto-detect true, auto-download true)

This error will be resolved when you install JDK 7 on your machine. I assume this error is caused by gradle since Java 7 support is discontinued and gradle could not install it automatically (auto-download is enabled). I will track this issue on #38.